src/components/--Footer/Footer.js

```diff
@@ -1,35 +1,81 @@
+import { Button, Footer as GrommetFooter, Grommet, Nav, Text } from "grommet";
+import { Facebook, Linkedin, Mail, Twitter } from "grommet-icons";
 import React from "react";
-import PropTypes from "prop-types";
-import styled from "styled-components";
-
-import { Text } from "../--Typography";
-import Logo from "../--Logo";
-import SocialLinks from "../--SocialLinks";
 
 /**
  * Footer component
  *
  */
 
-const StyledFooter = styled.div`
-  height: 50px;
-  display: flex;
-  justify-content: space-between;
-  align-items: baseline;
-`;
+const socialLinks = [
+  {
+    title: "Email",
+    icon: <Mail />,
+    href: "mailto:mail@varya.me",
+  },
+  {
+    title: "Twitter",
+    icon: <Twitter />,
+    href: "https://twitter.com/varya_en",
+  },
+  {
+    title: "Linkedin",
+    icon: <Linkedin />,
+    href: "https://www.linkedin.com/in/varyastepanova/",
+  },
+  {
+    title: "Facebook",
+    icon: <Facebook />,
+    href: "http://www.facebook.com/varvara.stepanova.9",
+  },
+];
+
+const theme = {
```

Comment:
For nav buttons I need redefine hover/active style here, without affecting the rest of buttons. Maybe it can be moved to global styles, depending on how many types of buttons we need (Grommet has default/primary/secondary)

According[ this ](https://github.com/grommet/grommet/issues/3049#issuecomment-486469949) therer are multiple ways to customize the style within a compobnent. This way with custom theme also used in their [ storybook ] (https://storybook.grommet.io/?path=/story/controls-button--custom).
But we can also rewrite it to use styled components wrappers. What do you think?